sharding/testcases/ledger/ledger-testcase.md

```diff
@@ -0,0 +1,99 @@
+# 账本测试用例
+
+## 账本介绍
+Shard中会执行root区块和shard区块，因此有root和shard两个账本。
+为什么需要root账本？
+
+* 每个shard可以有不同version／height的root账本，保证分片中的状态一致性
+* 实现shard账本与root block height状态的绑定
+
+## 测试用例
+
+### case1, shard基本信息保存在rootshard上
+shard的基本信息包括shardID、shard的creator、shard的状态state、shard创建时父shard的区块高度、shard的配置信息以及参与该shard共识的的peer信息。
+
+#### 步骤
+1. 构造创建shard的交易
+2. 将交易发往rootshard 等待交易落账
+3. 构建查询shardInfo的交易
+4. 将该交易分别发往rootshard和子shard
+
+#### 预期
+
+发往rootshard交易能够查询到子shard的基本信息
+
+发往子shard的交易执行失败
+
+### case2, 首次启动分片
+
+#### 步骤
+
+1. 创建好shard, 激活该shard
+2. 启动shard， 记录下启动shard时对应的rootshard 区块高度
+3. 查看账本的变化(Chain文件的变化)
+
+#### 预期
+
+shard会先启动同步线程，同时生成rootshard Chain文件，此时shard并不会出块，也就不会有shard账本，待同步到启动shard的区块高度时，shard开始出块，shard ledger开始生成
+
+
+### case3, rootshard推出来的子shard事件消息(分片治理相关的事件)
+rootshard推出来的shard治理事件消息保存在rootshard的eventstore中。

```

Comment:
shard chain sync root chain block info,save some shard gov event to shard parent ledger?